index.html

```diff
@@ -10,24 +10,25 @@
 </head>
 
 <body>
-  <h1>Awesome Books</h1>
+
+  <h1>All AwesomeBooks</h1>
 
   <div id="book-list">
   </div>
 
-  <form class="form">
-    <input class="field" type="text" id="title" placeholder="Title">
-
-    <input class="field" type="text" id="author" placeholder="Author">
-    <input class="btn" type="submit" value="add-book" id="add">
-
-  </form>
-
-
+  <section class="inputs">
+    <hr>
+    <h2>Add a new title</h2>
+    <form class="form">
+      <input class="field" type="text" id="title" placeholder="Title">
 
+      <input class="field" type="text" id="author" placeholder="Author">
```

Comment:
- [ ] Kindly make the input fields required, so that users would have to fill both inputs before submitting data. 👍